query/src/test/scala/hmda/query/serialization/InstitutionViewProtobufSerializerSpec.scala

```diff
@@ -11,7 +11,7 @@ class InstitutionViewProtobufSerializerSpec extends PropSpec with PropertyChecks
 
   val institutionList = Gen.listOf(institutionGen)
   property("InstitutionViewState must serialize to protobuf and back") {
-    forAll(institutionList, Gen.choose(0l, 100000l)) { (i, seq) =>
+    forAll(institutionList, Gen.choose(0: Long, 100000: Long)) { (i, seq) =>
```

Comment:
This is fine, for for next time, `0L` and `100000L` would suffice. The problem with using lowercase is that it is harder to read as a letter if you are not using an IDE